src/libs/algolia/fetchAlgolia/buildAlgoliaParameters/tests/buildGeolocationParameter.test.ts

```diff
@@ -104,7 +104,7 @@ describe('buildGeolocationParameter', () => {
     expect(result).toEqual(expectOutput)
   })
 
-  it('should return geolocation parameter for "all" location type', () => {
+  it('should return geolocation parameter for everywhere location type with "all" around radius', () => {
```

Comment:
'should return geolocation parameters with aroundradius as "all" if locationFilter is everywhere'

Reply:
C'est pas forcément all pour tous les everywhere du coup si tu regardes en dessous.
En lisant ta proposition j'ai l'impression que si je suis everywhere je suis forcément à all

Reply:
hum effectivement, mais là je ne comprend pas bien l'assertion telle qu'elle est car dans le code après y a pas de location Type. Le comportement attendu ce serait quoi tu crois ? C'est pour tester le around radius ?

Reply:
En fait si tu utilises le param `aroundRadius` et que tu es en everywhere, on l'utilise sinon on maintient le `all`

Reply:
hum pas sure de bien comprendre, tant pis on peut en rester là s'il faut hotfixé :)